data/interface.go

```diff
@@ -445,4 +445,6 @@ type ShardHeaderExtendedHandler interface {
 	ShardHeaderHandler
 	GetIncomingMiniBlockHandlers() []MiniBlockHandler
 	SetIncomingMiniBlockHandlers(miniBlockHandlers []MiniBlockHandler) error
+	GetHeaderHandler() HeaderHandler
+	GetIncomingEventHandlers() []EventHandler
```

Comment:
Not needed a method for set, like:
SetIncomingEventHandlers(eventHandlers []EventHandler) error 
?

Reply:
No need for it.
We only create this header in one place using non-abstract/interface object. 
In other places is only needed to get data from this 'proof' in order to validate it

Reply:
ok